notebooks/407-pedestrian-tracking-webcam/407-pedestrain-tracking.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.            self.output_layer = self.compiled_model.output(0)
</pre><p>output layer is not changed from inference to inference. This code can be moved to constructor</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>